manifests/d/DinoNuhagic(nuhi)/nLite/1.4.9.3/DinoNuhagic(nuhi).nLite.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: DinoNuhagic(nuhi).nLite
+PackageVersion: 1.4.9.3
+PackageLocale: en-US
+Publisher: Dino Nuhagic (nuhi)
+PackageName: nLite
+License: Dino Nuhagic (nuhi)

```

Comment:
A publisher name is not a license; Please provide actuall license terms such as `Proprietary`, `MIT`, `GPL-2.0`, or other terms as applicable